assignments/array-methods.js

```diff
@@ -56,28 +56,76 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+
+// for (let i = 0; i < runners.length; i++) {
+//     fullName.push(runners[i].first_name + ” ” + runners[i].last_name); //for loop that accomplishs the task.
+// }
+
+runners.forEach((a)=> {
+   fullName.push(`${a.first_name} ${a.last_name}`);
+});
+
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
-let allCaps = [];
-console.log(allCaps); 
+
+let allCaps = runners.map((a) => {
+ return (a.first_name.toUpperCase());
+});
+
+
+console.log(allCaps);
+
 
 // ==== Challenge 3: Use .filter() ====
 // The large shirts won't be available for the event due to an ordering issue.  Get a list of runners with large sized shirts so they can choose a different size. Return an array named largeShirts that contains information about the runners that have a shirt size of L and log the result
-let largeShirts = [];
+let largeShirts = runners.filter((a) => {
+  return a.shirt_size === "L";
```

Comment:
:+1: Good work using the `a` to represent each runner and selecting the property off of that object.